bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/transport/LegacyPort.java

```diff
@@ -496,6 +494,7 @@ private void initDevice(InsteonAddress a, @Nullable LegacyDevice device) {
         public void initialize() {
             try {
                 Msg msg = Msg.makeMessage("GetIMInfo");
+                addListener(this);
```

Comment:
This seems unrelated to the HTTP client refactoring, or maybe I'm missing something? What is the consequence of this not being here?

Reply:
No consequence. The listener is only needed when the first message is sent to the modem. It didn't need to be added when the `LegacyPort` object is initialized.

Reply:
I missed that it was previously in the constructor. Thanks!